lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -59,7 +59,8 @@ def type
     end
 
     def valid_attestation_statement?
-      @attestation_type, @attestation_trust_path = attestation_statement.valid?(authenticator_data, client_data.hash)
+      @attestation_type, @attestation_trust_path, @aaguid =
+        attestation_statement.valid?(authenticator_data, client_data.hash)
```

Comment:
Do we really need the `aaguid` to be returned as part of `#valid?`?
Can't we just have it be a separate method that gets the AAGUID from the attestation statement?

I'm even asking myself now why don't we have the same for `attestation_type` and `attestation_trust_path`. Possible I am forgetting a strong reason why we went with that approach :-)

Reply:
Yes, after thinking about this a bit more I agree. I'll change it to attr_reader for AAGUID, AAID and attestationKeyIdentifier.

The current return types are a very literal implementation of [the standard](https://www.w3.org/TR/webauthn/#attestation-formats) 😄 I can change that too.

> The procedure returns either:
> - An error indicating that the attestation is invalid, or
> - An implementation-specific value representing the attestation type, and the trust path. This attestation trust path is either empty (in case of self attestation), an identifier of an ECDAA-Issuer public key (in the case of ECDAA), or a set of X.509 certificates.

Reply:
:smile: :+1: